service/informer/informer_test.go

```diff
@@ -490,7 +491,8 @@ func Test_Informer_ensureProject(t *testing.T) {
 			ExpectedUpdated: true,
 		},
 
-		// Test 4
+		// Test 4 that a different ref causes a deployment update of the right
+		// project.
```

Comment:
Nitpick but this comment missed ensures.

Reply:
Fixed.